### PR TITLE
Added some responsiveness to Quick. May need to tune it.

### PR DESCRIPTION
**Pull request by oyta**
Edit: The first commit was adding some responsiveness to the page so that we can utilize more of the display real estate on more devices.

### CODE REVIEW

**Comment by oyta** [comment]
Should have started a new PR? Anyways, the latest commit adds timezone support to the chart. The Python code returns number of minutes offset from UTC and Highchart uses that info to show the correct timestamp.

**Comment by pgdr** [comment]
> Should have started a new PR?

No, that's fine.  It's less than 150 lines, so we should be able to review it.  If you give me some time, I will do exactly that.

File: sensor/models.py [hunk 1]
```diff
@@ -253,7 +253,10 @@ def create(cls , time = None, format = None, locale = False ):
     @classmethod
     def now(cls):
         return timezone.now()
-
+    
+    @classmethod
+    def timezoneOffset(cls):
+        return -1*timezone.localtime(timezone.now()).utcoffset().seconds/60
```
Comment (pgdr) [inline sensor/models.py]: what is this?

**Comment by oyta** [comment]
It calculates the timezone offset from UTC to the timezone in the Django
Settings. Maybe it can be done smoother, however I did not find a better
way.

The -1 is to make the offset negative going east of UTC as needed by
Highcharts.

timezone.now() returns the UTC-time and timezone.localtime() returns the
time in the timezone in Django settings.

The utcoffset() returns a timedelta object that only holds days, seconds
and microseconds, thereof the convertion to hours that is needed in
Highcharts.
tor. 2. mar. 2017 kl. 12.46 skrev Pål Grønås Drange <
notifications@github.com>:

**Comment by joakim-hove** [comment]
@pgdr : merge?

**Comment by pgdr** [comment]
merged and deployed.

**Pull request merged**